Frameworks/GLKit/GLKMath.mm

```diff
@@ -735,27 +732,57 @@ GLKIT_EXPORT GLKMatrix3 GLKMatrix4GetMatrix3(GLKMatrix4 m) {
 }
 
 /**
-   @Status Caveat
-   @Notes Only works on orthonormal transforms.
+   @Status Interoperable
 */
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3Invert(GLKMatrix3 m, BOOL* isInvertible) {
-    UNIMPLEMENTED();
-
-    // This is only going to work in very limited circumstances.
-    // (ie, m is an orthonormal transform).
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3Invert(GLKMatrix3 matrix, bool *isInvertible) {
```

Comment:
Thanks for adding these.  Could you please add the tests for these methods in the unit test file so we know these work and won't regress in the future?